test/org/sagebionetworks/bridge/models/schedules/IntervalTaskSchedulerTest.java

```diff
@@ -0,0 +1,457 @@
+    package org.sagebionetworks.bridge.models.schedules;
+
+import static org.junit.Assert.assertEquals;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleTestUtils.dt;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleTestUtils.assertDates;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleType.ONCE;
+import static org.sagebionetworks.bridge.models.schedules.ScheduleType.RECURRING;
+
+import java.util.List;
+import java.util.Map;
+
+import org.joda.time.DateTime;
+import org.junit.Before;
+import org.junit.Test;
+
+import com.google.common.collect.Maps;
+
+public class IntervalTaskSchedulerTest {
+
+    private Map<String, DateTime> events;
+    private List<Task> tasks;
```

Comment:
Tasks? To spare me from typing List<Task> tasks several dozen times. I can null it in a @Before method but the class is instantiated each time a test is run.